halyard-cli/src/main/java/com/netflix/spinnaker/halyard/cli/command/v1/NestableCommand.java

```diff
@@ -71,6 +72,20 @@ public void setColor(boolean color) {
 
   private static JobExecutor jobExecutor;
 
+  private static String[] failureMessages = {
+    "I'm sorry " + System.getProperty("user.name") + ", I'm afraid I can't do that.",
+    "If at first you don't succeed, try again with a different set of flags.",
+    "Looks like you're trying to deploy Spinnaker. Would you like some help?"
+  };
+
+  private static void showRandomFailureMessage() {
+    if (ThreadLocalRandom.current().nextInt(0, 100) < 5) {
+      int index = ThreadLocalRandom.current().nextInt(0, failureMessages.length);
```

Comment:
Whenever we're inside that if statement, roll will be a random int between 5 and 0. Now index is no longer uniformly distributed, and will land on 0 and 1 twice as often as 2 given that there are 3 messages. That's why I regenerate the random number.